internal/server/handlers.go

```diff
@@ -619,6 +626,16 @@ func (a *API) UpdateIdentityInfoFromProvider(c *gin.Context) error {
 	return access.UpdateUserInfoFromProvider(c, info, user, provider)
 }
 
+// validateProvider checks that a provider being modified is valid
+func (a *API) validateProvider(c *gin.Context, provider *models.Provider) error {
+	oidc, err := a.providerClient(c, provider, "http://localhost:8301")
```

Comment:
This url is ignored because validate doesn't look at it, is that right? Or is this url being used?

I was a bit surprised by it, so maybe a comment about it being ignored?

It'd be nice if we could avoid needing to use it. I'm not sure if that's possible. Maybe not worth the refactor right now.

Reply:
This was confusing, updated to an empty redirect URL and a comment